src/bosh.js

```diff
@@ -16,6 +16,10 @@
                 core.$build
             );
         });
+    } else if (typeof exports === 'object') {
+        var wrapper = require('./core');
```

Comment:
Would you please rename `wrapper` to `core`, like it's called in the AMD section above. I know you chose `wrapper` because that's what's used elsewhere (so it's more my fault for using the word `wrapper` too much) however I find it a bit confusing here.

The same applies to `src/websocket.js`.

Other than that, looks good! I'll merge immediately afterwards.

Thanks.

Reply:
Done :)
Thank you